README.md

```diff
@@ -1 +1,22 @@
-# STM32Ethernet
\ No newline at end of file
+# STM32Ethernet
+
+## Ethernet Library for Arduino
+
+With an STM32 board with Ethernet compatibility, this library allows an STM32
+board (NUCLEO, DISCOVERY, ...) to connect to the internet.
+
+For more information about this library please visit us at
+http://www.arduino.cc/en/Reference/Ethernet
+
+## Note
+The library is based on LwIP, a Lightweight TCP/IP stack.
+http://git.savannah.gnu.org/cgit/lwip.git
+
```

Comment:
Mention that LWIP has been ported as Arduino lib and this lib is dependent of this one